frontend/__tests__/middleware.test.ts

```
@@ -0,0 +1,131 @@
+    const encoded1 = encoder1.encode(TEST_SECRET)
+    const encoded2 = encoder2.encode(TEST_SECRET)
+
+    expect(encoded1).toEqual(encoded2)
+  })
+
+  it('should meet minimum length requirement', () => {
+    expect(TEST_SECRET.length).toBeGreaterThanOrEqual(32)
+  })
+})
+
+export { createTestToken, TEST_SECRET }
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)